src/main/java/net/irisshaders/lilybot/commands/support/UpdateDrivers.java

```diff
@@ -11,41 +10,96 @@
 import java.time.Instant;
 
 public class UpdateDrivers extends SlashCommand {
-    public UpdateDrivers() {
-        this.name = "update-drivers";
 
+    public UpdateDrivers() {
+        this.name = "drivers";
         this.help = "Informs the user on how to update drivers.";
-
         this.defaultEnabled = true;
-
         this.guildOnly = false;
-
-        this.botPermissions = new Permission[]{Permission.MESSAGE_WRITE};
-
-        this.botMissingPermMessage = "The bot can't write messages!";
+        this.children = new SlashCommand[]{
+                new AMDDrivers(),
+                new NvidiaDrivers(),
+                new IntelDrivers()
+        };
     }
 
     @Override
     protected void execute(SlashCommandEvent event) {
+        // Ignored because all commands are sub-commands
+    }
+
+    public static class AMDDrivers extends SlashCommand {
+
+        public AMDDrivers() {
+            this.name = "radeon";
```

Comment:
Radeon should be capitalized and changed to Radeon/AMD or just AMD

Reply:
Names cannot be capitalized

Reply:
Doesn't need to be, but AMD is the company behind them , which is what we have for all the others, with Radeon being the brand